app/models/hint.rb

```diff
@@ -1,5 +1,36 @@
+require 'stringex/core_ext'
```

Comment:
If you don't want to auto-require, you can `require: false` in Gemfile.

https://stackoverflow.com/questions/4800721/bundler-what-does-require-false-in-a-gemfile-mean